### PR TITLE
First  visualisation for tree v2

### PR DESCRIPTION
**Pull request by Giska**

### CODE REVIEW

File: packages/react-craft-ai-decision-tree/src/components/decisionTree.jsx [hunk 1]
```diff
@@ -9,38 +9,56 @@ const DecisionTreeContainer = styled('div')`
...
+const DecisionTree = ({
+  scale,
+  position,
+  data,
+  height,
+  width,
+  updateSelectedNode,
+  updatePositionAndZoom
+}) => {
+  const treeVersion = data._version.split('.')[0];
```
Comment (cloderic) [inline packages/react-craft-ai-decision-tree/src/components/decisionTree.jsx]: Using https://yarnpkg.com/en/package/semver would be more resilient

File: packages/react-craft-ai-decision-tree/src/utils/utils.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  COLOR_LEAVES_CONFIDENCE_1
+} from './constants';
+
+export function computeLeafColor(confidence) {
+  const blend = Math.pow(confidence, 3);
+  return mix(blend, COLOR_LEAVES_CONFIDENCE_1, COLOR_LEAVES_CONFIDENCE_0);
+}
+
+export function findSelectedNode(path, tree) {
+  let pathArray = path.split(';');
```
Comment (cloderic) [inline packages/react-craft-ai-decision-tree/src/utils/utils.js]: extract the separator in a constant file.

File: packages/react-craft-ai-decision-tree/src/components/edges.jsx [hunk 3]
```diff
@@ -80,15 +87,21 @@ class Edges extends React.Component {
       .attr('transform', (d) => `translate(${d.x},${d.y})`)
       .remove();
 
-    // Update the links…
+    // Update the links… (they are ordred)
     let link = svg.selectAll('path.link')
-      .data(links, (d) => d.target.id);
+      .data(links, (d) => {
+        d.linkClass = defaultLinksCssClass;
```
Comment (cloderic) [inline packages/react-craft-ai-decision-tree/src/components/edges.jsx]: Add a css class when selected to make styleable from the outside.

File: packages/react-craft-ai-decision-tree/src/components/nodeInformation/statistics.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Statistics = ({ node }) => {
+  if (node.nb_samples) {
+    return (
+      <div className='node-predictions'>
+        <h3 style={{ textAlign: 'center' }}>Statistics</h3>
```
Comment (cloderic) [inline packages/react-craft-ai-decision-tree/src/components/nodeInformation/statistics.jsx]: Better to use emotion tags than inline styles

File: packages/react-craft-ai-decision-tree/stories/decisionTree.js [hunk 5]
```diff
@@ -282,3 +325,77 @@ storiesOf('Tree in content', module)
...
+    ])
+  )
+  .add('tree', () => (
+    <DecisionTree
+      width={ number('Width', 600, sizeBoundOptions) }
+      height={ number('Height', 500, sizeBoundOptions) }
+      data={ tree }
+    />
+  ))
+  .add('Tree v1 node information - root node', () => (
```
Comment (cloderic) [inline packages/react-craft-ai-decision-tree/stories/decisionTree.js]: Capture d’écran 2019-03-15 à 09 43 31

File: CHANGELOG.md [hunk 6]
```diff
@@ -5,15 +5,28 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/craft-ai/react-craft-ai-components/compare/v0.0.18...HEAD) ##
+## [Unreleased](https://github.com/craft-ai/react-craft-ai-components/compare/v0.0.18...HEAD)
 
-## [0.0.18](https://github.com/craft-ai/react-craft-ai-components/compare/v0.0.17...v0.0.18) - 2019-02-28 ##
+## Added
+
+- Diplay the number of samples on the leaves
```
Comment (Totoketchup) [inline CHANGELOG.md]:
```suggestion
- Display the number of samples on the leaves
```

**Pull request merged**